bugbot/rules/not_landed.py

```diff
@@ -263,37 +263,18 @@ def get_bz_userid(self, phids):
             return {}
 
         def handler(user, data):
-            data[str(user["id"])] = user["name"]
+            data[str(user["id"])] = (user["name"], user["nick"])
```

Comment:
Assigning the user dict instead of creating a tuple will improve the readability:
```suggestion
            data[str(user["id"])] = user
```

In this case, we may want to rename the method from `get_bz_userid` to `get_bz_users`.

Reply:
renamed method